buildconfig/pygame-stubs/image.pyi

```diff
@@ -0,0 +1,11 @@
+from typing import Optional, Tuple, List, Union
+
+from pygame.surface import Surface
+
+
+def load(filename: str, namehint: Optional[str]='') -> Surface: ...
+def save(surface: Surface, filename: str) -> None: ...
+def get_extended() -> bool: ...
+def tostring(surface: Surface, format: str, flipped: Optional[bool]=False) -> str: ...
+def fromstring(string: str, size: Union[List[int], Tuple[int, int]], format: str, flipped: Optional[bool]=False) -> Surface: ...
```

Comment:
image.c@image_fromstring does not use keyword arguments.
Is there a way to specify list of only 2 elements [int, int]?

Reply:
Sadly not (or at least I didn't find any way): https://stackoverflow.com/questions/44833822/specify-length-of-sequence-or-list-with-python-typing-module. That is why I put both list and tuple, and not for example sequence (so users will at least get warning if they use tuple).